shared/testgrid/testgrid.go

```diff
@@ -38,7 +38,11 @@ const (
 
 // jobNameTestgridURLMap contains harded coded mapping of job name: Testgrid tab URL relative to base URL
 var jobNameTestgridURLMap = map[string]string{
-	"ci-knative-serving-continuous": "knative-serving#continuous",
+	"ci-knative-serving-continuous": 					"serving#continuous",
+	"ci-knative-serving-istio-1.0.7-mesh": 		"serving#istio-1.0.7-mesh",
+	"ci-knative-serving-istio-1.0.7-no-mesh": "serving#istio-1.0.7-no-mesh",
+	"ci-knative-serving-istio-1.1.2-mesh": 		"serving#istio-1.1.7-mesh",
```

Comment:
shouldn't this be `ci-knative-serving-istio-1.1.7-mesh`?

Reply:
yep, leftover from testing. fixed now.